controls/raddatetimepicker/getting-started.md

```diff
@@ -92,6 +93,21 @@ Normally you would want to bind the __SelectedValue__ property to your data as d
 	<telerik:RadDateTimePicker SelectedValue="{Binding Established}" />
 {{endregion}}
 
+## Manually Updating the Bound Property
+
+In case the bound property value needs to be manually updated at some state, the __DateTimeText__ property of the control can simply be set to be equal to the __CurrentDateTimeText__ one.
```

Comment:
Can we add one or two sentences of what the purpose of e each of the properties is? Here's the information from the summaries of the properties:
"The DateTimeText property changes while user has typed a text in the textbox. 
There is a difference between DateTimeText and CurrentDateTimeText. 
DateTimeText property changes when user had already entered a text. For example, this happens when the textbox has lost the focus. 
CurrentDateTimeText property changes every time when user is typing in the textbox."